api/src/pcapi/core/offerers/api.py

```diff
@@ -613,6 +615,10 @@ def validate_offerer(offerer: models.Offerer, author_user: users_models.User) ->
                 "Could not send validation confirmation email to offerer",
                 extra={"offerer": offerer.id},
             )
+    for managed_venue in managed_venues:
+        if managed_venue.adageId:
+            emails = get_emails_by_venue(managed_venue)
+            send_eac_offerer_activation_email(managed_venue, list(emails))
```

Comment:
```suggestion
            send_eac_offerer_activation_email(managed_venue, list(emails))
            break
```
Dans le cas (improbable) ou il y aurait plusieurs venues avec des adageId on n eveut envoyer le mail qu'une seule fois. Donc une fois qu'on a envoyer le mail on sort de la boucle.